docs/immortal-empires/skaven.md

```diff
@@ -26,3 +26,31 @@
     * Kraka Drak, Goromadny Tribe and Wintertooth
   * **Reward:**
     * _Tainted-Foul Flesh_: Mutagens: +26 per turn, Weapon strength: +13% for all Clan Moulder units
+
+## Deathmaster Snikch: Clan Eshin
+
+* **Route I - Service of the Grand Nightlord**
+  * Perform Ritual: Plunge into Anarchy
+  * Assassinate 13 enemy Agents
+  * Perform 13 Eshin Actions
+  * **Reward:**
+    * _Eshin Trials_: Recruit Rank +13 for Assassins, action success chance +13% for Assassins, Assassinate action success chance +13% for Assassins, +52 Melee AP damage for Assassins
+
+* **Route II - Assassin**
+  * Defeat Ku'gath Plaguefather in battle with Deathmaster Snikch's army
+  * Defeat Belegar Ironhammer in battle with Deathmaster Snikch's army
+  * Defeat Malus Darkblade in battle with Deathmaster Snikch's army
+  * **Rewards:**
+    * _A Plague No More_: +25% Technology research rate, -1 turn wound recovery time for all characters
+    * _Even Iron Can Shatter_: -40 Diplomacy with all Dwarven factions, +20 leadership vs all Dwarven factions
```

Comment:
Make it -39 Diplomacy
+20 Leadership looks crazy, +13 is already huge enough. I will give it something like +6/+8, but I prefer to keep alive the running joke